query-engine/js-connectors/src/proxy.rs

```diff
@@ -33,11 +33,14 @@ pub(crate) struct CommonProxy {
 /// This is a JS proxy for accessing the methods specific to top level
 /// JS driver objects
 pub(crate) struct DriverProxy {
-    start_transaction: AsyncJsFunction<Option<String>, JsTransaction>,
+    start_transaction: AsyncJsFunction<(), JsTransaction>,
 }
 /// This a JS proxy for accessing the methods, specific
 /// to JS transaction objects
 pub(crate) struct TransactionProxy {
+    /// transaction options
+    options: TransactionOptions,
```

Comment:
I feel what you're saying, but I'd wait to have both stable WebSocket and HTTP adapters (e.g., for Neon), and then re-assess whether this property can actually be defined once per each connector, or whether it could remain as a transaction property. Would that be ok?

Reply:
Alternatively, this refactoring could be implemented by a follow-up PR.